t/proto_template/buf.gen.yaml

```diff
@@ -4,7 +4,7 @@ version: 'v1'
 managed:
   enabled: true
   go_package_prefix:
-    default: 'github.com/REPLACE_GITHUB_ORG_NAME/REPLACE_GITHUB_REPO_NAME/gen/go'
+    default: 'github.com/REPLACE_GITHUB_ORG_NAME/REPLACE_GITHUB_REPO_NAME'
```

Comment:
This was a bug. We upload a go module _within_ the `gen/go` path. Therefore when importing protos as a consuming application, this suffix will not be a part of the path.

Reply:
Easy fix